fractal_server/app/schemas/user.py

```diff
@@ -118,9 +118,13 @@ class UserCreate(schemas.BaseUserCreate):
 
     # Validators
 
-    _slurm_accounts = validator("slurm_accounts", allow_reuse=True)(
-        val_unique_list("slurm_accounts")
-    )
+    @validator("slurm_accounts")
+    def slurm_accounts_validator(cls, value):
+        for i, element in enumerate(value):
+            valstr(attribute=f"slurm_accounts[{i}]")(element)
```

Comment:
My understanding is that `valstr` does not only act as a validator (raising errors if needed) but it also strips leading/trailing space characters. Would it be doable to have this functionality in this function as well?

Reply:
Now fixed. Thanks!